### ISSUE TITLE
"Easter Egg Day" is broken, and prevents the game from loading on the 1st of April and 9th of December

### ISSUE THREAD

**Issue by andrewjbennett** [iChun/BetterThanLlamas#3]
You've got a bug with "Easter Egg Day", which prevents any Minecraft installs with your BetterThanLlamas mod from being able to load the game on the 1st of April and 9th of December.

The error message looks like this:

```
net.minecraftforge.fml.ModLoadingException: Better Than Llamas (betterthanllamas) encountered an error during the done event phase
§7java.lang.NullPointerException: Cannot invoke "net.minecraft.world.level.Level.m_46658_()" because "p_21369_" is null
```

I can attach the full log if you want to debug it, but I'm just going to recommend that people remove your mod (and will use this issue as the canonical link).

**Comment by Ashy99** [comment]
[This mod has had issues with its Easter Egg days for years](https://github.com/iChun/BetterThanLlamas/issues/2 ), with no official fix or way to disable the events. Encouraging people to not use this mod sounds like the best option here.

**Comment by iChun** [comment]
There was a fix for this but it was 1.19+ only. This has been fixed for 1.18 now and the version is out on CurseForge. Sorry about the crashes!

**Comment by Twisted-Code** [comment]
Heh, personally I didn't mind too much, at least once I realized the solution was to simply remove the mod for 24 hours. 
Although I do have to ask: you did test this right? Like set your system clock to the trigger dates? it seems like it was crashing for EVERYONE so would've been obvious in your dev environment unless something changed that caused it to break or regress.
